tools/azure-sdk-tools/ci_tools/functions.py

```diff
@@ -434,9 +437,130 @@ def build_whl_for_req(req: str, package_path: str) -> str:
         logging.info("Building wheel for package {}".format(parsed.name))
         create_package(req_pkg_path, temp_dir, enable_sdist=False)
 
-        whl_path = os.path.join(temp_dir, find_whl(parsed.name, parsed.version, temp_dir))
+        whl_path = os.path.join(temp_dir, build_temp_whl(parsed.name, parsed.version, temp_dir))
         logging.info("Wheel for package {0} is {1}".format(parsed.name, whl_path))
         logging.info("Replacing dev requirement. Old requirement:{0}, New requirement:{1}".format(req, whl_path))
         return whl_path
     else:
         return req
+
+
+def find_sdist(dist_dir: str, pkg_name: str, pkg_version: str) -> str:
+    """This function attempts to look within a directory (and all subdirs therein) and find a source distribution for the targeted package and version."""
+    # This function will find a sdist for given package name
+    if not os.path.exists(dist_dir):
+        logging.error("dist_dir is incorrect")
+        return
+
+    if pkg_name is None:
+        logging.error("Package name cannot be empty to find sdist")
+        return
+
+    pkg_name_format = "{0}-{1}.zip".format(pkg_name, pkg_version)
+    pkg_name_format_alt = "${0}-{1}.tar.gz"
```

Comment:
This looks like it could be vestigial from `tar.gz` support investigation?

Reply:
I'm just writing to support it so we don't need to handle it later 👍